styles/weather.css

```diff
@@ -0,0 +1,3 @@
+.container .row div .weather-button{
+	margin: 10px 0 10px 0;
```

Comment:
You can use shorthand css `10px 0` instead of `10px 0 10px 0`. See the section labelled Syntax here: https://developer.mozilla.org/en/docs/Web/CSS/margin